zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/weapon/gun/energy/resolute
+	name = "DSS ECAR \"Resolute\""
+	desc = "\"DeepFlare Security System\" energy carbine. The byzantine internal capacitor design requires this gun to fire in bursts of three. It has notably become a go to fire-arm for station security forces with it's stun capabilities."
```

Comment:
It has notably become a ***go-to firearm*** for station security ***due to its*** stun capabilities.

Reply:
Other than this, the rest is fine. Just fix the description and it'll be good to go

Reply:
Fixed